From 4eb086c5e7469d86d28c0d6bd3c66ef0a26dd7a1 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 3 Jun 2024 15:04:42 -0700
Subject: [PATCH 4/4] Fix AJANTV2 includes in backport patch

---
 cmake/Modules/FindLibAJANTV2.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/Modules/FindLibAJANTV2.cmake b/cmake/Modules/FindLibAJANTV2.cmake
index 2ae1801..50c24d3 100644
--- a/cmake/Modules/FindLibAJANTV2.cmake
+++ b/cmake/Modules/FindLibAJANTV2.cmake
@@ -136,11 +136,11 @@ if(LIBAJANTV2_FOUND)
       ${AJA_LIBRARIES_INCLUDE_DIR} ${AJA_LIBRARIES_INCLUDE_DIR}/ajaanc ${AJA_LIBRARIES_INCLUDE_DIR}/ajabase
       ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2 ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/includes)
   if(CMAKE_HOST_SYSTEM_NAME MATCHES "Windows")
-    list(APPEND LibAJANTV2_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/win)
+    list(APPEND AJA_LIBRARIES_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/win)
   elseif(CMAKE_HOST_SYSTEM_NAME MATCHES "Darwin")
-    list(APPEND LibAJANTV2_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/mac)
+    list(APPEND AJA_LIBRARIES_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/mac)
   elseif(CMAKE_HOST_SYSTEM_NAME MATCHES "Linux")
-    list(APPEND LibAJANTV2_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/lin)
+    list(APPEND AJA_LIBRARIES_INCLUDE_DIRS ${AJA_LIBRARIES_INCLUDE_DIR}/ajantv2/src/lin)
   endif()
 
   set(LIBAJANTV2_LIBRARIES ${AJA_NTV2_LIB})
-- 
2.45.1

